packages/plugin-ext/src/plugin/document-data.ts

```diff
@@ -80,7 +81,7 @@ export class DocumentDataExt {
             this._document = {
                 get uri(): theia.Uri { return that.uri; },
                 get fileName(): string { return that.uri.fsPath; },
-                get isUntitled(): boolean { return that.uri.scheme === 'untitled'; },
+                get isUntitled(): boolean { return that.uri.scheme === UNTITLED_SCHEME; },
```

Comment:
Same issue as above. You can use `Schemes.untitled` instead (imported from `uri-components`).

Reply:
In order to avoid unnecessary influence, roll back the code for the time being. Thank you again for your help @msujew @colin-grant-work